drivers/planar/clarity_matrix_spec.cr

```diff
@@ -0,0 +1,10 @@
+DriverSpecs.mock_driver "Planar::ClarityMatrix" do
+  # on connect it should do_poll the device
+  should_send("op A1 display.power ? \r")
```

Comment:
When writing specs, you also want to mock mock the device responses so that you test full request -> response behaviour. For some info on this, see: https://github.com/PlaceOS/drivers/blob/master/docs/writing-a-spec.md#testing-streaming-io.